### PR TITLE
{bio}[foss/2023b] BWA v0.7.17

### PR DESCRIPTION
**Pull request by grpiccoli**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/BWA/BWA-0.7.17-foss-2023b.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+#
+# Version >= 0.7.17
+# Author: Adam Huffman, Big Data Institute, University of Oxford
+#
+# Note that upstream development is mainly at: https://github.com/lh3/bwa
+##
+name = 'BWA'
+version = '0.7.17'
+homepage = 'http://bio-bwa.sourceforge.net/'
```
Comment (verdurin) [inline easybuild/easyconfigs/b/BWA/BWA-0.7.17-foss-2023b.eb:20]:
```suggestion
homepage = 'https://bio-bwa.sourceforge.net/'
```
Looks like `https` works now, at last.

File: easybuild/easyconfigs/b/BWA/BWA-0.7.17-foss-2023b.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+description = """Burrows-Wheeler Aligner (BWA) is an efficient program that aligns
+ relatively short nucleotide sequences against a long reference sequence such as the human genome."""
+toolchain = {'name': 'foss', 'version': '2023b'}
```
Comment (verdurin) [inline easybuild/easyconfigs/b/BWA/BWA-0.7.17-foss-2023b.eb:23]: I think some of the blank lines between sub-sections have been lost, which makes it less readable.

Please add a blank line here before `toolchain`, and another after `toolchainopts`.

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by verdurin** [comment]
@grpiccoli Welcome to EasyBuild, and thanks for your contribution.
I've added a couple of comments, with suggestions for small fixes.

**Comment by grpiccoli** [comment]
Hi @verdurin, thank you for your welcome and feedback.
I just pushed a corrected version using --update-pr.
Cheers

**Review by grpiccoli — COMMENTED** [review]
Thank you for your feedback

**Review by grpiccoli — COMMENTED** [review]
approved

**Comment by grpiccoli** [comment]
Hi @verdurin,

I deleted the original file which was using foss 2023b and switch it for foss 2023a, so you will not see your changes reflected on the file that was deleted but the new file instead. I also added you as a contributor,

Thanks

**Comment by grpiccoli** [comment]
Closing this PR and opening a fresh one, Cheers

**Pull request closed without merging**